packages/engine/src/avatar/components/TransparencyDitheringComponent.ts

```diff
@@ -54,14 +53,16 @@ export const TransparencyDitheringComponent = defineComponent({
   onInit: (entity) => {
     return {
       ditheringDistance: 0.35,
-      ditheringExponent: 5
+      ditheringExponent: 5,
+      position: new Vector3()
     }
   },
 
   onSet: (entity, component, json) => {
     if (!json) return
     if (matches.number.test(json.ditheringDistance)) component.ditheringDistance.set(json.ditheringDistance)
     if (matches.number.test(json.ditheringExponent)) component.ditheringExponent.set(json.ditheringExponent)
+    if (matchesVector3.test(json.position)) component.position.set(json.position)
```

Comment:
we should not be copying vector3 by reference, instead by value, and updating that value in a system. if we don't do this, we run the risk of mutating the upstream data accidentally

Reply:
let's also call this `center`

Reply:
> we should not be copying vector3 by reference, instead by value, and updating that value in a system. if we don't do this, we run the risk of mutating the upstream data accidentally

Agree, I do not intend on keeping the current reference to the camera's position in the component.